guide/07-working-with-bigdata/02-making-your-data-accessible-to-ga-server.ipynb

```diff
@@ -0,0 +1,838 @@
+{
```

Comment:
<p>In this example, the spatial reference of the dataset is set to 4326, but we know this data is from New Orleans, Louisiana, and is actually stored in the <a href="https://spatialreference.org/ref/esri/102682/html/" target="_blank">Louisiana State Plane Coordinate System</a>. We need to edit the manifest with the correct spatial reference: {"wkid": 102682, "latestWkid": 3452}. Knowing the location of this data and the coordinate system to which it belongs, we will edit our manifest. This will set the correct spatial reference.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1076/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>